extensions/resource-deployment/notebooks/sql2019-image-docker.ipynb

```diff
@@ -0,0 +1,72 @@
+﻿{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.6.6",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": "First, pull the Microsoft SQL Server 2019 Image",
```

Comment:
have you thought about how to localize this content?

Reply:
good point, i will make the path for notebook localizable so that we can pick up notebook for a specific language.